tests/cpgrid/adapt_cpgrid_test.cpp

```diff
@@ -33,24 +33,14 @@
 
 #define BOOST_TEST_MODULE AdaptTests
 #include <boost/test/unit_test.hpp>
-#include <boost/version.hpp>
-#if BOOST_VERSION / 100000 == 1 && BOOST_VERSION / 100 % 1000 < 71
-#include <boost/test/floating_point_comparison.hpp>
-#else
-#include <boost/test/tools/floating_point_comparison.hpp>
-#endif
```

Comment:
I could need see where this was moved. Is it not needed  anymore. What are we using instead of
```
BOOST_TEST(c1[c] == c2[c], boost::test_tools::tolerance(1e-12));
```
then?